src/main/java/duke/utility/parser/Parser.java

```diff
@@ -0,0 +1,256 @@
+package duke.utility.parser;
+
+import duke.duke_exception.DukeException;
+import duke.tasklist.TaskList;
+import duke.tasklist.task_types.Deadline;
+import duke.tasklist.task_types.Event;
+import duke.tasklist.task_types.Task;
+import duke.tasklist.task_types.ToDo;
+import duke.utility.ui.UiMessage;
+
```

Comment:
Consider removing the blank line at line 10 to ensure consistency across code files..